### PR TITLE
Move sidechains networks to fed peg project and use FedPeg consensus options in them

### PR DESCRIPTION
**Pull request by noescape00**

### CODE REVIEW

**Comment by codingupastorm** [comment]
hey @noescape00 why do this?

File: src/Stratis.Features.FederatedPeg/Networks/CirrusV2/CirrusMain.cs [hunk 1]
```diff
@@ -53,26 +54,26 @@ internal CirrusMain()
...
                 maxBlockBaseSize: 1_000_000,
                 maxStandardVersion: 2,
                 maxStandardTxWeight: 100_000,
                 maxBlockSigopsCost: 20_000,
                 maxStandardTxSigopsCost: 20_000 / 5,
-                federationPublicKeys: federationPubKeys,
+                genesisFederationMembers: genesisFederation,
                 targetSpacingSeconds: 16,
-                votingEnabled: false,
+                votingEnabled: false, // Does not work on sidechains yet. Sidechains need to handle removal of fed member and recreation of multisig first.
```
Comment (codingupastorm) [inline src/Stratis.Features.FederatedPeg/Networks/CirrusV2/CirrusMain.cs]: Don't we need voting enabled for whitelisting to work?

I believe we agreed that we're not going to worry about the recreation of the multisig for the moment and would still allow adding / removing members?

**Comment by noescape00** [comment]
>hey @noescape00 why do this?

Hm. I've forgot what we decided the last time we discussed this. 

Do we want to move the networks in or we want consensus options to be in networks project? 
I'd go with first option but I don't mind doing it ether way.

File: src/Stratis.Features.FederatedPeg/Networks/CirrusV2/CirrusMain.cs (hunk 1 above)
Comment (noescape00) [reply src/Stratis.Features.FederatedPeg/Networks/CirrusV2/CirrusMain.cs]: >Do we need voting enabled for whitelisting to work?

Yes, it's required. 

I think what we also can do is enable voting but disable voting for adding\removing fed members. Or we might leave it enabled for all cases and deal with potential consequences of not recreating the multisig.

**Comment by noescape00** [comment]
I'll recreate the PR without moving the networks to fedpeg. Instead will move fedpeg poa consensus options to the sidechains networks project

**Pull request closed without merging**